great_expectations/datasource/data_connector/configured_asset_glue_catalog_data_connector.py

```diff
@@ -0,0 +1,140 @@
+from great_expectations.core.batch_spec import GlueCatalogBatchSpec
+from great_expectations.datasource.data_connector.configured_asset_sql_data_connector import (
+    ConfiguredAssetSqlDataConnector,
+)
+from great_expectations.execution_engine import ExecutionEngine, SparkDFExecutionEngine
+
+try:
+    import boto3
+except ImportError:
+    boto3 = None
+
+logger = logging.getLogger(__name__)
+
+
+class ConfiguredAssetGlueCatalogDataConnector(ConfiguredAssetSqlDataConnector):
+    """
+    Extension of ConfiguredAssetSqlDataConnector used to connect to AWS Glue Data Catalog
+
+    The ConfiguredAssetGlueCatalogDataConnector is one of two classes (InferredAssetGlueCatalogDataConnector
+    being the other one) designed for connecting to data through AWS Glue Data Catalog.
+
+    A ConfiguredAssetGlueCatalogDataConnector requires an explicit listing of each DataAsset you want to
+    connect to. This allows more fine-tuning, but also requires more setup. You will need define to the
+    database and table names.
+    """
+
+    def __init__(
+        self,
+        name: str,
+        datasource_name: str,
+        execution_engine: Optional[ExecutionEngine] = None,
+        assets: Optional[Dict[str, dict]] = None,
+        boto3_options: Optional[dict] = None,
+        batch_spec_passthrough: Optional[dict] = None,
+    ):
+        """
+        A DataConnector that requires explicit listing of AWS Glue Catalog tables you want to connect to.
+
+        Args:
+            name (str): The name of this DataConnector
+            datasource_name (str): Name of datasource that this DataConnector is connected to
+            execution_engine (ExecutionEngine): Execution Engine object to actually read the data
+            assets (dict): dict of asset configuration
+            boto3_options (dict): optional boto3 options
+            batch_spec_passthrough (dict): dictionary with keys that will be added directly to batch_spec
+        """
+        logger.warning(
+            "Warning: great_expectations.datasource.data_connector.ConfiguredAssetGlueCatalogDataConnector is "
+            "experimental. Methods, APIs, and core behavior may change in the future."
+        )
+        super().__init__(
+            name=name,
+            datasource_name=datasource_name,
+            execution_engine=execution_engine,
+            assets=assets,
+            batch_spec_passthrough=batch_spec_passthrough,
+        )
+
+        if boto3_options is None:
+            boto3_options = {}
+
+        try:
+            self._glue = boto3.client("glue", **boto3_options)
+        except (TypeError, AttributeError):
+            raise ImportError(
+                "Unable to load boto3 (it is required for ConfiguredAssetGlueCatalogDataConnector)."
+            )
+
+    @property
+    def execution_engine(self) -> SparkDFExecutionEngine:
+        return cast(SparkDFExecutionEngine, self._execution_engine)
+
+    def _update_data_asset_name_from_config(
```

Comment:
@lccasagrande On one hand, this implementation of `_update_data_asset_name_from_config()` adds some robustness to the one from the parent class.  On the other hand, it is missing some of the provisions, such as prefix/sufficx and schema name directives, provided by the parent class.  Might we simple be able to augment the `_update_data_asset_name_from_config()` implementation in the parent class with your check for `data_asset_config` and then simply not override this method?  Please let me know; if affirmative, I can fix it on our end, or you can make the change as part of your pull request (please let me know your preference).  Thank you.

Reply:
I did a code refactoring and I am not using the **ConfiguredAssetSqlDataConnector** as parent class anymore. I am follow the standard approach, inheriting from **DataConnector** class.